### DISCUSSION TITLE
is it possible to add change the curve of an edge ?

### DISCUSSION THREAD

**Discussion by Jasminou** [xyflow/xyflow#2219] [Q&A] [answered]
Hi I wanted to know if its possible to apply a specfic value to change the curve of an edge ex: the step edge want them to be less rounded then squares.
thanks

**Comment by moklick** [comment, marked as answer]
Hey @Jasminou 

it's possible! You need to create a custom edge and use the [`getSmoothStepPath` helper function](https://reactflow.dev/docs/api/edges/edge-utils/#getsmoothsteppath). It has a `borderRadius` option.
